src/components/views/rooms/RoomTile.tsx

```diff
@@ -102,6 +104,9 @@ export default class RoomTile extends React.PureComponent<IProps, IState> {
         };
         this.notificationState = RoomNotificationStateStore.instance.getRoomState(this.props.room);
         this.roomProps = EchoChamber.forRoom(this.props.room);
+        if (this.props.resizeNotifier) {
+            this.props.resizeNotifier.on("middlePanelResizedNoisy", this.onResize);
```

Comment:
We should probably use `middlePanelResized` as we don't need sub-200ms responsiveness on this. Resizing fires a *LOT*

Reply:
Yes. I will change that.